pkg/providers/cloudstack/validator_test.go

```diff
@@ -16,9 +16,10 @@ import (
 )
 
 const (
-	testClusterConfigMainFilename        = "cluster_main.yaml"
-	testClusterConfigMainWithAZsFilename = "cluster_main_with_availability_zones.yaml"
-	testDataDir                          = "testdata"
+	testClusterConfigMainFilename            = "cluster_main.yaml"
+	testClusterConfigMainAutoscalingFilename = "cluster_main_autoscaling.yaml"
```

Comment:
I think this file is missing. Also, is it necessary to add a new file? Can't you just take the existing one to be parsed and modify it in the test after it's parsed?